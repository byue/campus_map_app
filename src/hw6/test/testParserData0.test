# test parser on a graph with 0 nodes

LoadGraph heroes data0.tsv
ListChildren heroes a
FindPath heroes a b