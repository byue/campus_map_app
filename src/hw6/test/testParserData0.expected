# test parser on a graph with 0 nodes

loaded graph heroes
nodes are not present in graph
unknown character a
unknown character b